.travis.yml

```diff
@@ -1,23 +1,21 @@
 language: python
-sudo: false
+
 python:
   - 2.7
-  - 3.4
   - 3.5
   - 3.6
+  - 3.7
+  - 3.8-dev
   - pypy
   - pypy3
-matrix:
-    include:
-        - python: "3.7"
-          dist: xenial
-          sudo: true
+
 script:
   - coverage run -m zope.testrunner --test-path=src  --auto-color --auto-progress
 
 after_success:
   - coveralls
-notifications:
+
+  notifications:
   email: false
```

Comment:
This looks like an editing mistake that introduces a YAML syntax error.  Let's fix it.

Reply:
Thank you for the hint.